src/main/resources/db/migration/V55__Add_missing_documents_record_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE missing_documents_record (
+          id UUID PRIMARY KEY,
+          recall_id UUID NOT NULL REFERENCES recall (id),
+          email_id UUID NOT NULL REFERENCES document (id),
+          detail text NOT NULL,
+          version INTEGER NOT NULL,
+          created_by UUID NOT NULL,
+          created_date_time TIMESTAMPTZ NOT NULL
+);
+
+CREATE TABLE missing_document_category (
+    missing_document_record_id    UUID    NOT NULL REFERENCES missing_documents_record (id),
+    document_category   VARCHAR(32) NOT NULL,
+    UNIQUE (missing_document_record_id, document_category)
```

Comment:
I haven't spotted a test verifying this constraint.  For documents we have those in the IntegrationTest.  Tests named like "fun `cannot save and flush ... throws DataIntegrityViolationException`()".

Reply:
we are saving a set for this. But for other restraints I will add.